app/assets/javascripts/views/users.js

```diff
@@ -0,0 +1,37 @@
+App.Views.Users = Backbone.View.extend({
+  el: 'section',
+  template: JST['templates/users'],
+
+  initialize: function(options) {
+    this.users = options.users;
+
+    this.data = {
+      highestPrivilege: App.currentUserRoles.highestPrivilege()
+    };
+
+    this.$el.html(this.template());
+    this.$users = $('.panel-group');
+  },
+
+  render: function() {
+    if (this.data.highestPrivilege === 'admin') {
```

Comment:
Just App.currentUserRoles.highestPrivilege() === 'admin' looks more descriptive in this case.